sbin/tpm2-evt-log-parser.awk

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/gawk -bf
```

Comment:
tpm2_eventlog can parse the event log from files easily. IIUC because there is no utility for parsing TPM 1.2 event log, we used awk for it. And to be consistent we use awk too for TPM2.0?

Reply:
It doesn't like TXT special entries:

```
ERROR: PCR Index 255 is out of bounds for max available PCRS 32
ERROR: failed to parse tpm2 eventlog
ERROR: Unable to run tpm2_eventlog
```

Also, every event type is printed as `Unknown event type` which doesn't say anything.

Reply:
I see.